backend/src/v2/driver/driver.go

```diff
@@ -671,13 +671,42 @@ func resolveInputs(ctx context.Context, dag *metadata.DAG, iterationIndex *int,
 				}
 			}
 			value, hasValue := inputs.GetParameterValues()[name]
+			/*
```

Comment:
NIT: remove dead code?

Reply:
Thank you, this was kept for testing purposes. I will remove this later.